sample.config.toml

```diff
@@ -9,8 +9,10 @@
 [templates]
   default = ''
 
-[remotes]
-  default = { name = '', url = '' }
-
 [srs]
   default = ''
+
+# resources created before all tests and deleted at the end.
+[neededResources]
```

Comment:
After thought `preCreatedResources` is more significant. Because all config resources are needed.